connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/processors/bop/BOPRequestPreparer.java

```diff
@@ -66,25 +65,26 @@ public void process(final Exchange exchange) {
             Log.infof("Recipients: %s", Arrays.toString(recipients.toArray()));
         }
 
+        // Prepare the email to be sent.
         final Email email = new Email(
             subject,
             body,
             recipients
         );
 
-        JsonObject bopBody;
-        if (emailConnectorConfig.isSkipBopUsersResolution()) {
-            final SendEmailsRequest request = new SendEmailsRequest();
-            request.addEmail(email);
-            bopBody = JsonObject.mapFrom(request);
-        } else {
-            final Emails emails = new Emails();
-            emails.addEmail(email);
-            bopBody = JsonObject.mapFrom(emails);
-        }
```

Comment:
I did this because I wasn't sure `SendEmailsRequest` would work with MBOP/Go because of the new fields. A&M said in Slack that unknown fields wouldn't trigger any error in MBOP, so I guess it's safe to remove `Emails` as you did in this PR.

Reply:
I am preparing a patch for the MBOP project which makes it support the parameters too, I think I'll be able to deliver it today.